scripts/src/submission/submission.py

```diff
@@ -188,7 +188,7 @@ class Report:
 @dataclass
 class Source:
     found: bool = False
-    path: str = None  # Path to the Chart.yaml
+    path: str = None
```

Comment:
I'd like to see an example of what this path is expected to look like in a comment.

You haven't touched other code in this PR, so I'll just mention it as an improvement opportunity here.

If it's an absolute path, calling the attribute "full_path" may not be out of scope either, but it isn't required to accomplish what I'd look for in the comment.

Reply:
I've added some docstrings and comments